.github/workflows/unit-and-integration-tests-unified.yml

```diff
@@ -16,10 +16,20 @@
 name: Unit & Integration tests CI
 on:
   push:
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
```

Comment:
I think the `examples` folder used for setting up hadoop for ITs - should we ignore changes in that or let it run the full CI? The changes in `examples` folder are also very in-frequent.

Reply:
You are right. fixed that too.